Representable/representable
# fix geojson export crs + right-hand rule

**changes**
- fixed #271 : geojson export so that it doesn't get crs errors or right-hand rule errors

**to test**
- go to submission page for a community
- click export geojson
- upload to https://geojsonlint.com/ or related tool
- confirm that there are no errors
